### PR TITLE
Removing slash from variable names

### PR DESCRIPTION
**Pull request by eric-czech**
Changes for https://github.com/pystatgen/sgkit/issues/81.

This should work after https://github.com/pystatgen/sgkit/pull/83 goes through.

### CODE REVIEW

File: sgkit_plink/pysnptools.py [hunk 1]
```diff
@@ -122,13 +121,13 @@ def coerce_code(v, codes):
...
     names = [f[0] for f in BIM_FIELDS]
-    df = dd.read_csv(str(path) + ".bim", sep=sep, names=names, dtype=BIM_DF_DTYPE)
+    df = dd.read_csv(str(path), sep=sep, names=names, dtype=BIM_DF_DTYPE)
     df["contig"] = df["contig"].where(df["contig"] != "0", None)
     return df
 
 
 def read_plink(
-    path: PathType,
+    path: Union[PathType, Tuple[PathType, PathType, PathType]],
```
Comment (ravwojdyla) [inline sgkit_plink/pysnptools.py]: This change is unrelated to slashes, also has unaddressed issue from #11

**Review by ravwojdyla — CHANGES_REQUESTED** [review]

File: sgkit_plink/pysnptools.py (hunk 1 above)
Comment (eric-czech) [reply sgkit_plink/pysnptools.py]: I'll merge the changes from https://github.com/pystatgen/sgkit-plink/pull/11 and https://github.com/pystatgen/sgkit-plink/pull/15 whenever they go through.

**Comment by eric-czech** [comment]
@ravwojdyla, @tomwhite, or @jeromekelleher can one of you approve this now?

**Review by ravwojdyla — APPROVED** [review]

**Comment by eric-czech** [comment]
ty
